### PR TITLE
Added the news post for Playtest 20150808

### PR DESCRIPTION
**Pull request by Mailaender**
https://github.com/OpenRA/OpenRAWeb/pull/243 with just a linkfix to the download section and the playtest ID updated.

### CODE REVIEW

**Comment by penev92** [comment]
:+1:

File: content/news/playtest-20150808.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+We are pleased to announce the second playtest on the road to our next stable release.  We have now entered the feature freeze phase, which means that we have stopped adding new features and instead focus on fixing any bugs that you might find.  That means that we need your help with testing, testing and some more testing!  Please report any bugs and problems you encounter [on our bug tracker](http://bugs.openra.net). Thank you for helping to improve OpenRA!
```
Comment (Phrohdoh) [inline content/news/playtest-20150808.md]: Please use the Oxford comma: `testing, testing, and some more testing!`
`bugs or problems` please.

File: content/news/playtest-20150808.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* Paratroopers can now be selected and given orders before they land.
+* Two new single-player missions have been added, Soviets 04a and 04b.
+* Single-player missions have been polished, and Allies 05 has received a new ultra-hard difficulty.
+* Mission replays can again be viewed through the replay browser.
+* A warning message is now given when a server admin selects a map with custom rules.
+* Range circles are now visible to allied players and spectators.
+* Players can now hide the in-game UI (using a hotkey, bound to Ctrl+Shift+H by default).
+* We now support installing game content from *The First Decade* DVD.
+
+Of interest for modders and mappers will be the addition of new Lua API functions.  Disguising spies, capturing or infiltrating units and buildings using scripting is now possible.  Other newly available functions can be used to play arbitrary sound files, flash the screen and control the new palette effect trait.
```
Comment (Phrohdoh) [inline content/news/playtest-20150808.md]: `sound files, flash the screen, and control the new palette effect trait` please.

File: content/news/playtest-20150808.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Of course we did not only add new things, but also fixed bugs and annoyances.  Here's a small sample:
+
+* A number of crashes have been resolved.
+* The score screen can now be closed to have a look at the battlefield after the match.
+* The 'Options' button will no longer blink for no reason in skirmish maps.
```
Comment (Phrohdoh) [inline content/news/playtest-20150808.md]: s/for no reason/without reason/ it sounds more professional.

File: content/news/playtest-20150808.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+* The duration bar on chronoshifted units in the Red Alert mod is no longer visible to enemy players.
+* The aircraft and helicopter cruise altitude in the Red Alert mod the has been increased again.
```
Comment (Phrohdoh) [inline content/news/playtest-20150808.md]: Was it increased, or was it finally reset to what it was before decreasing it?
If the latter we should find a way to say that.

File: content/news/playtest-20150808.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+* The unit duplication crate bonus has been removed from the Tiberian Dawn mod.
+
+For all the gritty details, check out the [full changelog](http://changelog.openra.net).
```
Comment (Phrohdoh) [inline content/news/playtest-20150808.md]: Unnecessary comma.

File: content/news/playtest-20150808.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+Grab the installer for this playtest [from our download page](/download/) and give it a try!
+
+<div style="text-align:center" markdown="1">
+
+![Custom map rule warning](/images/news/20150808-cnc-lobby.png)
+
+The game will warn players in the lobby when the host selects a map with custom rules.  This should help combat the small number of dishonest players who have been creating cheat maps (shame on you!).
```
Comment (Phrohdoh) [inline content/news/playtest-20150808.md]: I don't like the second sentence here at all.
They aren't *cheat maps* they are maps that alter the rules which we fully allow so it is valid.
That does not make players dishonest either.

I get that it is playful, but I'm not a fan of coming off like this.

**Comment by Mailaender** [comment]
@obrakmann Do you want to be credited for this post?

**Comment by Mailaender** [comment]
Updated with @Phrohdoh's feedback.

**Comment by obrakmann** [comment]
On 2015-08-08 14:13, Matthias Mailänder wrote:
> @obrakmann <https://github.com/obrakmann> Do you want to be credited for
> this post?

Yes, of course.

**Comment by Unit158** [comment]
:+1: I already did enough nitpicking :grin:

File: content/news/playtest-20150808.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+For all the gritty details check out the [full changelog](http://changelog.openra.net).
+
+Grab the installer for this playtest [from our download page](/download/) and give it a try!
+
+<div style="text-align:center" markdown="1">
+
+![Custom map rule warning](/images/news/20150808-cnc-lobby.png)
+
+The game will warn players in the lobby when the host selects a map with custom rules.  This should help combat the small number of dishonest players, who adjusted rules to their advantage.
```
Comment (pchote) [inline content/news/playtest-20150808.md]: The `,` between players and who shouldn't be there.

File: content/news/playtest-20150808.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+Allied players and spectators can now see the range circles from units and buildings.
+
+<iframe width="600" height="333" src="https://www.youtube-nocookie.com/embed/3ai-MeYDx5c?rel=0" frameborder="0" allowfullscreen></iframe>
+
+New Lua API functions can be used to simulate simple weather effects.
+
+<iframe width="600" height="333" src="https://www.youtube-nocookie.com/embed/WoZoQILTaGk?rel=0" frameborder="0" allowfullscreen></iframe>
+
+Work continues on our Tiberian Sun mod.  Check out these clips from a recent test game!
```
Comment (pchote) [inline content/news/playtest-20150808.md]: For the sake of our inboxes I think we should add a disclaimer that the TS mod still isn't quite ready to be shipped by default.

Comment (pchote) [inline content/news/playtest-20150808.md]: I suggest `Work continues on our Tiberian Sun mod in preparation for a future release.<br/>Check out these clips from a recent test game!`

**Comment by Mailaender** [comment]
Updated again.

**Comment by pchote** [comment]
:+1:

**Pull request merged**